index.html

```diff
@@ -1,57 +1,57 @@
-                    </div>
-                </div>
-                <div class="ingredients-container">
-                    <div class="ingredients">
-                        <h3>Ingredients</h3>
-                        <img src="vegetarian-icon.svg" alt="vege-icon">
-                    </div>
-                    <div class="ingredients-text">
-                        <p>60 ml olive oil</p>
-                        <p>3 medium red onions, thinly sliced</p>
-                        <p>4 garlic cloves, thinly sliced</p>
-                        <p>1/2 teaspoon fine sea salt, plus more for seasoning</p>
-                        <p>1/2 teaspoon red pepper flakes</p>
-                        <p>3 tablespoons tomato paste</p>
-                        <p>50 g sun-dried tomatoes in oil, chopped</p>
-                        <p>2 tablespoons soy sauce</p>
-                        <p>400 g spaghetti, bucatini or bavette</p>
-                        <p>30 g chopped parsley</p>
-                        <p>Juice of 1/2 lemon</p>
-                        <p>60 g smoked farmer cheese</p>
-                    </div>
-                </div>
-            </div>
-            <div class="right-section">
-                <div class="preparation-container">
-                    <div class="preparation">
-                        <h3>Preparation</h3>
-                    </div>
-                    <div class="preparation-text">
-                        <p>Heat the oil over medium-low heat in a large skillet. Add the onion, garlic and salt. Cook, stirring frequently, until caramelized, about 15 minutes. Add the red pepper flakes, tomato paste, and sun-dried tomatoes. Cook stirring often, until the tomato paste becomes darker. Stir the soy sauce.</p>
-                        <p>Bring a pot of salted water to boil. Add the pasta and cook until it is al dente. Reserve 1 cup of pasta water  and then drain the pasta.</p>
-                        <p>In a small  bowl, combine the parsley and lemon juice and season of salt.</p>
-                        <p>Add the pasta and reserved pasta water to the skillet with tomato-onion mixture. Cook until sauce evenly coats the pasta.</p>
-                        <p>Divide the pasta among four bowls. Top with the parsley and smoked farmer cheese.</p>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </body>
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+        <meta charset="UTF-8">
+        <title>Recipe</title>
+    </head>
+    <body>
+        <div class="main-container">
+            <div class="left-section">
+                <div class="title-container">
+                    <div class="title">

```

Comment:
You don't need this container

Reply:
Do you mean left-section? In my opinion it should stay, as thanks to it, it is possible to put title and ingredients sections in column. When I removed this section, there was a total mess and I had no idea how to put everything in order as before.

Reply:
I mean `<div class="title">`

Reply:
removed